packages/flutter/lib/src/material/overscroll_painter.dart

```diff
@@ -0,0 +1,133 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter/material.dart';
```

Comment:
We're in the material package, so we don't want to import the material package itself.  (That's circular.)  Instead, we should import the specific files that we use.